RRTPlannerROS: {
  goal_tolerance: 0.30,

  rrt: {
    step: 0.20,
    min_num_nodes: 200,
    max_num_nodes: 3000,
  }
}